security/netbird/src/opnsense/service/conf/actions.d/actions_netbird.conf

```diff
@@ -1,17 +1,17 @@
 [start]
```

Comment:
file called actions_NETBIRD.conf, command called START means "netbird start" is correct.